app/models/vm/operations/guest.rb

```diff
@@ -1,4 +1,12 @@
 module Vm::Operations::Guest
+  extend ActiveSupport::Concern
+
+  included do
```

Comment:
Personal preference, I would rather see:
```ruby
def self.inclided(klass)
  klass.api_relay_method(:shutdown_guest)
  ...
end
```

But I understand if you're doing it for consistency and don't want to change it.